### PR TITLE
Feat/upload files

### PR DESCRIPTION
**Pull request by mitraws**
## Finalized upload functionality with library: Cloudinary

- User story: as a Specialists I want to view the idea and add input by uploading a document.
- Button added to a detailpage where a file can be uploaded
- Uploaded documents are saved in the Cloudinary account
- A preview of previous uploaded documents are displayed above the upload button
Upload1
Upload2

### CODE REVIEW

**Comment by mitraws** [comment]
Right now, they are not linked.

File: src/components/SpecialistPortal/Dashboard/CloudinaryService.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  cloudName: 'idealists',
+  format: 'json',
+  type: 'list',
+  version: Math.ceil(new Date().getTime() / 1000),
+};
+
+const urlPath = url(docTag.toString(), options);
+
+fetch(urlPath)
+.then(res => res.text())
```
Comment (dashakrolik) [inline src/components/SpecialistPortal/Dashboard/CloudinaryService.js:26]: indentation

File: src/components/SpecialistPortal/Dashboard/CloudinaryService.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  const cl = CoreCloudinary.new();
+  return cl.url(publicId, scOptions);
+};
+
+export const openUploadWidget = (options, callback) => {
+  const scOptions = Util.withSnakeCaseKeys(options);
+  window.cloudinary.openUploadWidget(scOptions, callback);
+};
+
+export async function  fetchDocs  (docTag, setter)  {
```
Comment (dashakrolik) [inline src/components/SpecialistPortal/Dashboard/CloudinaryService.js:15]: extra space after function

File: src/components/SpecialistPortal/Dashboard/FormInputIdeas.js [hunk 3]
```diff
@@ -6,11 +6,48 @@ import styled from "@emotion/styled";
...
+
 
 export default function IdeaDashboardDetail(props) {
   const [userIdeas, setUserIdeas] = useState([]);
   const [progress, setProgress] = useState([]);
+  const [docs, setDocs] = useState([]);
+
+  const docsSection = (
+    <section>
+    {docs.map(i => <Image
```
Comment (dashakrolik) [inline src/components/SpecialistPortal/Dashboard/FormInputIdeas.js:25]: Curious why are we using an image here?:)

File: src/components/SpecialistPortal/Dashboard/CloudinaryService.js (hunk 2 above)
Comment (mitraws) [reply src/components/SpecialistPortal/Dashboard/CloudinaryService.js:15]: Solved with prettier!

File: src/components/SpecialistPortal/Dashboard/CloudinaryService.js (hunk 1 above)
Comment (mitraws) [reply src/components/SpecialistPortal/Dashboard/CloudinaryService.js:26]: Solved with prettier!

File: src/components/SpecialistPortal/Dashboard/FormInputIdeas.js (hunk 3 above)
Comment (mitraws) [reply src/components/SpecialistPortal/Dashboard/FormInputIdeas.js:25]: This part of the code is for the preview of updated docs, according to the teacher this is necessary to display the image without refreshing the page.

**Pull request merged**